internal/storage_adapter.go

```diff
@@ -41,6 +43,14 @@ func (adapter *StorageAdapter) loadSettings() (map[string]string, error) {
 	return settings, nil
 }
 
+func (adapter *StorageAdapter) PrefixName() string {
```

Comment:
You used it only in the spare package "config".

Reply:
Removed this spare getter.